bundles/org.openhab.core.io.rest.auth/src/main/resources/OH-INF/config/config.xml

```diff
@@ -11,6 +11,17 @@
 			<description>Allow the use of Basic authentication to access protected API resources, in addition to access tokens
 				and API tokens.</description>
 		</parameter>
+		<parameter name="cacheExpiration" type="integer">
```

Comment:
I think it should also have a minimum ` min="0" ` ? Is it also possible to disable this cache by using "0" ?

Reply:
I've added `min="1"` (since it is not possible to deactivate the cache (on purpose)).

Reply:
You think we should add a deactivation? Value 0 might be misleading as @spacemanspiff2007 interpreted it as "cache does not expire".

Reply:
If it allows for disabling the cache without adding much other code it might be a nice addition for those concerned about security. The behavior should then be documented in the description.

Note that message digest algorithms like SHA-256 are also considered unsecure nowadays. That's because such hashes are relatively easy to compute and there is dedicated hardware that simplifies brute forcing them. E.g. classes to generate such hashes are deprecated in [Spring Security](https://github.com/spring-projects/spring-security/blob/aad76e68a2427ae525bc31dbb5ebe02905d6b6d3/crypto/src/main/java/org/springframework/security/crypto/password/MessageDigestPasswordEncoder.java#L78-L82).

Reply:
I'm aware that SHA-256 is a deprecated algorithm to **store** passwords.
But this is an in memory cache and get access to the hashed pw an attacker already needs to have memory access and/or code execution privileges.
If an attacker already has that, then imho brute forcing a basic auth hash is the least of the worries.

Reply:
@wborn SHA-256 is secure, the link to Spring Security just says that it isn't recommended for password hashes, since it is too fast to calculate - but this is exactly what we try to avoid here: Doing expensive calculations for users that already authenticated successfully. In our situation, there is no risk of any dictionary attacks here. See also https://github.com/traefik/traefik/issues/7897#issuecomment-780755004, which does the very same (with SHA-512 instead of SHA-256, but with the same arguments).

Reply:
I've added `0` as a valid configuration value to disable the cache.